### PR TITLE
Building font page

### PR DESCRIPTION
**Pull request by mugberto**
# Project: Using Bootstrap

This project is about using Bootstrap classes to create a responsive page that looks similar to the Newsweek.com home page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2394 @@
...
+                </h2>
+              </div>
+            </article>
+            <article class="col-md-3 card border-0 rounded-0 py-2 mb-3">
+              <div class="card-image-container position-relative mb-1">
+                <img
+                  src="./images/fe-privacy-qa-crystal-watson-03.jpg"
+                  class="card-img-top in-the-magazine-img rounded-0 mb-2"
+                  alt="Tedros and Xi Jimping"
+                />
```
Comment (lucasmazo32) [inline index.html:1299]: - [ ] The size of this articles are not the same as the original page. Kindly fix that.

> Original
image
> Your page
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2394 @@
...
+            </div>
+          </div>
+        </section>
+
+        <section class="in-the-magazine nw-bb bg-white container-fluid py-2 mb-3 mb-md-0">
+          <h1 class="section-heading-1 border-bottom-itm-1 nw-font-2 text-center px-md-3">
+            <span class="sect-icon nw-font-2 text-white nw-bg-red mr-2">N</span>
+            IN THE MAGAZINE
+          </h1>
+          <div class="row mx-md-0">
```
Comment (lucasmazo32) [inline index.html:1246]: - [ ] I am missing the sign up section. Please add it!

> image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

Great job. This page has many details and you almost copied it as it is. You need to make two small changes before the project gets approved. Please check the comments below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by mugberto** [comment]
> # Changes required 
> Great job. This page has many details and you almost copied it as it is. You need to make two small changes before the project gets approved. Please check the comments below.
> 
> ### Happy coding! 
> ## Lucas Mazo

Thank you for the review. I have fixed the issues.

**Review by christianotieno — CHANGES_REQUESTED** [review]
Hello @mugberto 👋 
##
## The [page](https://raw.githack.com/mugberto/newsweek-clone/building-font-page/index.html) is very responsive and shows good use of bootstrap.

For this project, however, we have to make sure nothing is messed up. Good job implementing the above changes.
-  The markup is required to not have any errors. Please validate it with this [tool](https://validator.w3.org/nu/#textarea) and fix the emerging errors. Thank you.

Make this change and then submit another code review. hank you.

**Comment by mugberto** [comment]
> Hello @mugberto 
> 
> ## The [page](https://raw.githack.com/mugberto/newsweek-clone/building-font-page/index.html) is very responsive and shows good use of bootstrap.
> For this project, however, we have to make sure nothing is messed up. Good job implementing the above changes.
> 
> * The markup is required to not have any errors. Please validate it with this [tool](https://validator.w3.org/nu/#textarea) and fix the emerging errors. Thank you.
> 
> Make this change and then submit another code review. hank you.

 I introduced these errors when I was fixing issues raised in the previous feedback. Thank you @christianotieno. I have taken notes. I will always validate before the submission.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project. :fireworks: Keep it up.
##
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job @mugberto! :+1:
- You can now merge to master. I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**